sunpy/net/tests/test_fido.py

```diff
@@ -97,6 +98,14 @@ def test_save_path():
             assert target_dir in f
             assert "eve{}0".format(os.path.sep) in f
 
+    with tempfile.TemporaryDirectory() as target_dir:
+        qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
+        path = pathlib.Path(os.path.join(target_dir, "{instrument}"+os.path.sep+"{level}"))
```

Comment:
Do `pathlib.Path` need to use `os.path.join`?
I had thought that pathlib.Path was meant to handle that.

Reply:
Missed it while copy-pasting!